frontend-react/src/pages/home/Sections/LiveMapSection.module.scss

```diff
@@ -0,0 +1,18 @@
+@use "../../../global-modules.scss" as *;
+
+.legend {
+    list-style: none;
+
+    li {
+        float: left;
+        margin-right: 8px;
```

Comment:
We need to align with USWDS' sizes as much as possible for margins/widths/heights/paddings/etc. Please update these styles to use USWDS' `u-` utilities with the nearest-matching size parameter to the design.